src/Subscription.php

```diff
@@ -732,6 +753,38 @@ public function updateQuantity(int $quantity, $invoiceNow = true)
         return $this;
     }
 
+    /**
+     * Schedule this subscription's quantity to be changed once the current cycle has completed.
+     *
+     * @param  int  $quantity
+     * @return $this
+     */
+    public function updateQuantityNextCycle(int $quantity)
+    {
+        throw_if(
+            $quantity < 1,
+            new LogicException('Subscription quantity must be at least 1.')
+        );
+        return DB::transaction(function () use ($quantity) {
```

Comment:
We'll also need to make sure the subscription doesn't get restarted if it has ended.